runtime/sam/op/merge/merge_test.go

```diff
@@ -96,7 +96,7 @@ func TestParallelOrder(t *testing.T) {
 	for i, c := range cases {
 		t.Run(strconv.Itoa(i), func(t *testing.T) {
 			zctx := zed.NewContext()
-			octx := &op.Context{Context: context.Background(), Zctx: zctx}
+			octx := &runtime.Context{Context: context.Background(), Zctx: zctx}
```

Comment:
This is one of two remaining occurrences of octx. You could rename it to rctx but I favor removing it and replacing `octx.Context` with `context.Background()` down below.
```suggestion
```